src/components/headers/BaseHeader.tsx

```diff
@@ -139,8 +145,8 @@ const BaseHeader = ({
   const compactContentAnimatedStyle = useAnimatedStyle(() =>
     hasCompactHeader
       ? {
-          opacity: interpolate(scrollY?.value || 0, [60, scrollEndThreshold], [0, 1], Extrapolate.CLAMP),
-          height: interpolate(scrollY?.value || 0, [60, scrollEndThreshold], [110, 95], Extrapolate.CLAMP)
+          opacity: interpolate(scrollY?.value || 0, [40, scrollEndThreshold], [0, 1], Extrapolate.CLAMP),
```

Comment:
NIT: Why change the initial interp value to 40 instead on 60? On iOS it makes the animation less smooth (we see the compact header being static before moving up).

Reply:
Ah shit, I wanted to do this only on Android! Good catch! I'll add it in my next PR